andela/healthchecks-prometheus
# Ft team allocation and department dashboard

#### What does this PR do?
This PR enables a team admin (user with team access enabled) to assign checks to invited team members.
A feature is also added to enable a team admin to create departments within a team. 
The team admin can also assign a check to a specific department. A drop down list is added to the checks dashboard to enable team users filter checks according to which departments they belong.
#### Description of Task to be completed?
- Add a departments form that can allow a team admin to add and remove departments
- Add a departments option in invite team form to specify to which department the invited member belongs
- Add an owner's option input to enable the admin assign a check to a certain team member
- Add a departments option input in the edit checks form for team admin to assign check to a specific department
- Add a drop down list to to enable a team member filter checks by department
#### How should this be manually tested?
- Using a user account with team access enabled create a two departments
- invite another user by specifying their email and specify their department by choosing one of the above
- Add 4 checks and assign 2 to the invited user
- The user should only be able to modify the checks assigned to the user
#### What are the relevant pivotal tracker stories?
#148476597
#148476599

#### Screenshots

<img width="1196" alt="add departments" src="https://user-images.githubusercontent.com/13675851/29351422-46b4b23a-826b-11e7-9e84-97026c2b641c.png">

<img width="646" alt="invite team member" src="https://user-images.githubusercontent.com/13675851/29351425-49af0918-826b-11e7-9b77-0197c3aed2ed.png">

<img width="197" alt="filter by department" src="https://user-images.githubusercontent.com/13675851/29351430-502b5774-826b-11e7-9875-0c163b00f4d5.png">

<img width="667" alt="aasign check to owner and department" src="https://user-images.githubusercontent.com/13675851/29351435-54fdb954-826b-11e7-93fd-56904b87894a.png">
